src/views/v2/top-bar/actions.php

```diff
@@ -14,16 +14,5 @@
  */
 ?>
 <div class="tribe-events-c-top-bar__actions">
```

Comment:
are we thinking of having multiple actions? or is this to future-proof in the case we add functionality? my questions are around why we didn't move this entire template to pro.

Reply:
I imagine themers/plugin authors may wish to add additional actions.